---
title: "Results"
order: 4
---
<!--The results of the experiment. See [Figure 1](#figure-1), [Figure 2](#figure-2), [Figure 3](#figure-3), and [Figure 4](#figure-4), for details.-->

### HIV proteins and structural predictors of evolution

The HIV genome encodes 15 proteins that can be divided into three general categories. The first, the enzymes, are all encoded by the *pol* gene which encodes three separate proteins. They are, in order of their reference genome numbering, protease, reverse transcriptase (including the RNase H domain), and integrase. Each of these three enzymes assumes a homodimeric quaternary structure and all three have clinically available inhibitory drugs. The second category of proteins encoded by the HIV genome are the structural proteins; they are encoded by two different genes in HIV. The matrix, capsid, and nucleocapsid are thought to be purely structural and are encoded by the *gag* gene. The external glycoprotein (gp120) and transmembrane glycoprotein (gp41) are encoded by the *env* gene, and serve the receptor-binding and uptake functions for the virus. The third category of HIV proteins carry out a disparate mix of accessory and regulatory functions for the virus. These include vpu, vif, vpr, p6, nef, rev, and tat. For this study, we focused on the enzymes and structural proteins. Specifically we analyzed sequence data and structures for protease, reverse transcriptase, integrase, the matrix and capsid proteins as well as the apical portion of the ectodomain of the spike glycoprotein, gp120. For each of these proteins, we wondered to what extent metrics taken entirely from the protein's structure could predict the evolution of the protein's sequence measured as $$ dN/dS $$. In addition, although structural metrics have been shown to predict evolution in several disparate proteins in unrelated viruses, it is still not known whether or not all of the proteins in a single virus would show a similar connection between structure and evolution.

There are a number of structure-based predictors of evolution (either measure as Rate4site or $$ dN/dS $$) that have been described previously ([Bustamante et al. 2000](http://www.ncbi.nlm.nih.gov/pubmed/10677853), [Franzosa and Xia 2009](https://dx.doi.org/10.1093/molbev/msp146), [Ramsey et al. 2011](https://dx.doi.org/10.1534/genetics.111.128025), [Scherrer et al. 2012](https://dx.doi.org/10.1186/1471-2148-12-179), [Shahmoradi et al. 2014](https://dx.doi.org/10.1007/s00239-014-9644-x), [Echave et al. 2014](https://dx.doi.org/10.1088/1478-3975/12/2/025002), [Huang et al. 2014](https://dx.doi.org/10.1186/1471-2148-14-78), [Marcos and Echave](https://dx.doi.org/10.7717/peerj.911); for a complete review see [Sikosek and Chan 2014](https://dx.doi.org/10.1098/rsif.2014.0419)); the most common are relative solvent accessibility (RSA) and weighted contact number (WCN). Both predictors have been shown to provide statistically significant predictions of $$ dN/dS $$ in other viral systems and large enzyme protein sets alike ([Sikosek and Chan 2014](https://dx.doi.org/10.1098/rsif.2014.0419), [Shahmoradi et al. 2014](https://dx.doi.org/10.1007/s00239-014-9644-x), [Meyer and Wilke 2015](http://dx.doi.org/10.1371/journal.ppat.1004940)); generally, WCN calculated on the amino acid side chain is a stronger predictor than RSA calculated in either the monomeric or functional multimeric state. However, for several viral proteins, RSA has been shown to be a better predictor of $$ dN/dS $$ ([Shahmoradi et al. 2014](https://dx.doi.org/10.1007/s00239-014-9644-x)), and it has been used successfully modeling the evolution of influenza ([Meyer and Wilke 2012](http://dx.doi.org/10.1093/molbev/mss217), [Meyer et al. 2013](http://dx.doi.org/10.1098/rstb.2012.0334)). Therefore, in this study, we used RSA as the baseline evolutionary predictor. In addition, it is possible to calculate RSA in any of several different conformational states. Most simply, one could ignore the higher order quaternary structure of protein complexes and measure RSA on each monomeric unit. Alternatively, one could generate ever more complex multimeric assemblies on which to calculate RSA. We chose to use the minimal multimeric state (called the biological assembly in the protein databank) which has been shown to be the strongest extensible predictor of $$ dN/dS $$ in influenza ([Meyer and Wilke 2015](http://dx.doi.org/10.1371/journal.ppat.1004940)). 

In addition to RSA, we used a single distance-based dataset to predict $$ dN/dS $$. This has previously proven effective at identifying regions of possible functional important in viral proteins ([Meyer and Wilke 2015](http://dx.doi.org/10.1371/journal.ppat.1004940)). In the case of HIV, we generated a set of distances using the pairwise distance from each C-alpha atom to every other C-alpha atom in the protein. Then, we performed a cross validation analysis to identify the best reference point in each protein while controlling for RSA.

### Protein-specific structural models of HIV evolution

**Evolution of the HIV capsid**

The capsid protein is a functional homo-hexamer with extensive contacts in adjacent monomeric units. The hexameric biological assemblies form the basic building blocks for constructing the entire super-structure of the viral capsid. We found that no part of the capsid protein appeared to be under particularly strong evolutionary pressure. With whole-gene $$ dN/dS = 0.2 $$ ([Table 1](#table-1)) the capsid has the second slowest rate of adaptation among the proteins we studied; also, the distribution of $$ dN/dS $$ shows almost no positively selected sites ([Figure 1A](#figure-1)). The combined RSA-Distance model could predict less than 10% of the variation in $$ dN/dS $$. The sites with the highest predicted rates of adaption were near the N-terminal region of the protein, and the sites with the lowest predicted rate of adaptation were near the C-terminal end ([Figure 2A](#figure-2)). As expected, the distance to sites nearest the N-terminal region of the monomeric protein showed the highest positive predictive power for $$ dN/dS $$ ([Figure 2B](#figure-2)); therefore, sites nearest the N-terminal region of the capsid were, on average, under the strongest pressure from structural constraints. 

**Evolution of the HIV *gp120* protein**

The HIV receptor-binding complex is made up of an ectodomain as well as transmembrane and intracellular segments. The ectodomain consists of a homo-trimer of hetero-dimers where each of the dimers consists of a *gp120* and gp41 protein. The *gp120* protein serves as the apical segment and the gp41 protein serves as the basal segment. Thus, *gp120* is farthest from the viral surface and the most exposed HIV protein to the host immune system. 

The apical solvent exposed surface of the *gp120* protein appears to be under intense adaptive pressure. In addition, with whole-gene $$ dN/dS = 0.8 $$ ([Table 1](#table-1)), *gp120* has the highest rate of adaptation in HIV and a large number of sites with $$ dN/dS > 1 $$([Figure 1B](#figure-1)). Therefore, solvent exposure alone accounted for a large fraction of the adaptive evolution ($$ R^{2} = 0.18 $$). Nevertheless, adding a single distance constraint to the model nearly doubled the variation explained. Where the RSA-only model produced a relatively uniform rate prediction over the entire surface of gp120, adding the set of distances to the linear model oriented the elevated $$ dN/dS $$ predictions toward the apical surface of the protein ([Figure 3A](#figure-3)). 

We also investigated the effect of *gp120* glycosylations on selection. We found that, even though glycosylation sites alone were able to predict a small amount of adaptive evolution in gp120, adding glycosylation sites to the combined model did not improve or change the overall rate predictions. This finding might be expected since glycosylation sites are on the protein surface. Thus, RSA, and not specific glycosylation sites, provides the most systematic adaptive pressure to *gp120* ([Figure 3B](#figure-3)). 

Finally, as we expected from the predicted $$ dN/dS $$ values, we found that the sites whose distances provided the highest positive predictive power were located on the apical surface of the ectodomain ([Figure 3C](#figure-3)). Therefore, sites near the surface of the protein and oriented farthest away from the viral surface are on average under the strongest adaptive pressure. Also, no clear connection emerged between the glycosylation pattern and the surface with maximum adaptive pressure ([Figure 3D](#figure-3)).

**Evolution of the HIV matrix**

The fundamental biological assembly of the HIV matrix consists of homo-trimers of the matrix protein. Unlike the other structural proteins (the capsid and the *gp120* protein), the matrix protein has a relatively small contact surface with adjacent subunits and a relatively high whole-gene $$ dN/dS = 0.5 $$ ([Table 1](#table-1)) and a large number of sites with $$ dN/dS > 1$$ ([Figure 1C](#figure-1)). Nevertheless, despite the structural differences in structure, shape, and whole-gene $$ dN/dS $$ between the matrix and capsid proteins, the predictors performed almost identically in the RSA-Distance combined models. The validated combined model for the matrix protein explained approximately 9.9% of the variation in $$ dN/dS $$, compared to 9.2% for the capsid. Also, like the capsid, there appeared to be no region that is under uniformly strong selective pressure ([Figure 4A](#figure-4)), and the cluster of sites with the highest pressure were on the long eccentric arms of the trimer ([Figure 4B](#figure-3)).

**Evolution of the HIV integrase**

The available structures of integrase show it as a symmetric homo-dimeric structure with long arms protruding from the interaction surface. No structural predictors provided a statistically significant model of integrase evolution ([Table 1](#table-1)). In addition, it showed the lowest rate of adaptive evolution and a very small number of sites with $$ dN/dS > 1 $$ ([Figure 1D](#figure-1).

**Evolution of the HIV protease**

The protease forms a heart-shaped homo-dimer. The catalytic site of the complex lies in the interface between the two subunits. In addition, the protease had a low average $$ dN/dS $$ ([Table 1](#table-1)) and a very small number of sites with $$ dN/dS > 1 $$ ([Figure 1E](#figure-1)). For functional reasons, the enzyme active site must be accessible, and the protein-protein binding region must maintain a relatively high solvent exposure ([Figure 5](#figure-5)) compared to other protein-protein interfaces. As a result, we expected the highly solvent exposed catalytic site to cause an RSA-only model to perform poorly. Indeed, RSA alone could explain only 4.7% of the variation in $$ dN/dS $$. However, adding distance produced the second best model among the six proteins tested. The $$ dN/dS $$ predictions with the combined RSA-Distance model showed that the catalytic site of protease is very conserved relative to the exterior regions of the protein ([Figure 5A](#figure-5)), a finding that would have been impossible without the distance predictor. Moreover, as with the *gp120* protein the pattern of predicted rates is almost perfectly recapitulated by the correlation map ([Figure 5B](#figure-5)).

**Evolution of the HIV reverse transcriptase**

As with the other HIV enzymes, reverse transcriptase has a relatively low rate of adaptation at $$ dN/dS = 0.25 $$ ([Table 1](#table-1)) and almost no sites with $$ dN/dS > 1 $$ ([Figure 1F](#figure-1)). The reverse transcriptase protein is structurally unique among the proteins in HIV. Although it forms a homo-dimeric quaternary structure, the two proteins in the functional assembly do not form the same tertiary structure. In addition, one of the two subunits in the functional assembly retains a separate catalytic domain known as RNase H. The unusual fact that the reverse transcriptase quaternary structure is composed on two identical polymers with $$ RMSD > 4.5~\overset{\circ}{A}$$ between their structures means that any structural model is likely doomed to failure. Despite this fact, we did find that the RSA of the first chain alone was a weak, but statistically significant predictor of $$ dN/dS $$. The weakness of the model was highlighted by the fact that the site-wise $$ dN/dS $$ predictions showed no clear trend toward important regions of the protein ([Figure 6A and 6B](#figure-6)). For example, the region contacting nucleic acid did not appear to be particular conserved. Also, no set of distances from the first chain could add any predictive power, and strangely the RNase H domain appeared to be strongly conserved relative to the nucleic acid contact site of the polymerase ([Figure 6C and 6D](#figure-6)).

### Summary of combined models of HIV evolution

In summary, we found that using RSA in a single predictor model performs poorly in the majority of HIV proteins ([Figure 7](#figure-7) and [Table 1](#table-1)). With the exception of gp120, RSA could predict less than 6% of the variation in $$ dN/dS $$. This is in contrast to influenza, where RSA explains greater than 10% of the variation in both the hemagglutinin and neuraminidase proteins ([Meyer and Wilke 2015](http://dx.doi.org/10.1371/journal.ppat.1004940)). In addition, for the integrase protein, RSA was not even a statistically significant predictor of $$ dN/dS $$; that makes integrase the only protein among the many hundred that have been tested to not display any folding-related evolutionary constraint. For the other five proteins, $$ dN/dS $$ could be at least partly predicted with RSA. 

Adding the distance to a reference point improved predictions of $$ dN/dS $$ in all of the proteins tested. On average, the models that included distance were able to account for nearly twice as much variation in $$ dN/dS $$ on the test data set than did the models with RSA alone. Importantly, our cross-validated test sets showed very similar $$ R^{2} $$ values to those from the training set. Therefore, our optimization scheme is likely not over-fitting the $$ dN/dS $$ data. Also, for each protein, there appeared to be a connection between the strength of the initial model that included only RSA and the strength of the best model that included both distance and RSA ([Figure 8](#figure-8)). However, protease was a significant outlier from the trend, and caused the overall correlation to not be significant. For example, the RSA-only model for protease was the third worst, but jumped to become the second best with a distance predictor. By contrast, the RSA-only model for reverse transcriptase was virtually identical in predictive power to that of protease; however, the addition of distance did not provide even a minimally significant improvement in $$ dN/dS $$ predictions. 
