<!-- This file should be ordered by appearance -->

<!-- Basic Tools -->
* Cock PJ, Antao T, Chang JT, Chapman BA, Cox CJ, Dalke A, Friedberg I, Hamelryck T, Kauff F, Wilczynski B, de Hoon MJ. 2009. [Biopython: freely available Python tools for computational molecular biology and bioinformatics](https://dx.doi.org/10.1093/bioinformatics/btp163). Bioinformatics 25: 1422-1423.

* Price MN, Dehal PS, Arkin AP. 2009. [FastTree 2 – approximately maximum-likelihood trees for large alignments](https://dx.doi.org/10.1371/journal.pone.0009490). PLOS ONE 5: e9490.

* Kosakovsky-Pond SL, Frost SDW, Muse SV. 2005. [HyPhy: hypothesis testing using phylogenies](https://dx.doi.org/10.1093/bioinformatics/bti079). Bioinformatics 21: 676-679.

* Muse SV, Gaut BS. 1994. [A likelihood approach for comparing synonymous and nonsynonymous nucleotide substitution rates, with application to the chloroplast genome](http://www.ncbi.nlm.nih.gov/pubmed/7968485). Mol. Biol. Evol. 11: 715-724.

* Yang Z. 2006. [Computational Molecular Evolution](http://download.bioon.com.cn/view/upload/month_0808/20080811_979a0656719f9157b466IruZP7mlp2U9.attach.pdf).

* Berman HM, Westbrook J, Feng Z, Gilliland G, Bhat TN, Wessig H, Shindyalov IN, Bourne PE. 2000. [The Protein Data Bank](https://dx.doi.org/10.1093/nar/28.1.235). Nucleic Acids Res. 28: 88-95.

* Berman HM. 2008. [The Protein Data Bank: a historical perspective](https://dx.doi.org/10.1107/S0108767307035623). Acta Crystallographica Section A: Foundations of Crystallography A64.

<!-- Structures -->
* Sarafianos SG, Das K, Tantillo C, Clark AD Jr, Ding J, Whitcomb JM, Boyer PL, Hughes SH, Arnold E. 2001. [Crystal structure of HIV-1 reverse transcriptase in complex with a polypurine tract RNA:DNA](https://dx.doi.org/10.1093/emboj/20.6.1449). EMBO J. 20: 1449-1461.

* Chen JC, Krucinski J, Miercke LJ, Finer-Moore JS, Tang AH, Leavitt AD, Stroud RM. 2000. [Crystal structure of the HIV-1 integrase catalytic core and C-terminal domains: a model for viral DNA binding](https://dx.doi.org/10.1073/pnas.150220297). Proc. Natl. Acad. Sci. 97: 8233-8238.

* Wang, JY,  Ling H,  Yang W,  Craigie R. 2001. [Structure of a two-domain fragment of HIV-1 integrase: implications for domain organization in the intact protein](https://dx.doi.org/10.1093/emboj/20.24.7333). EMBO J. 20: 7333-7343.

* Kim EE, Baker CT, Dwyer MD, Murcko MA, Rao BG, Tung RD, Navia MA. 1995. [Crystal structure of the HIV-1 integrase catalytic core and C-terminal domains: a model for viral DNA binding](https://dx.doi.org/10.1021/ja00108a056). J. Am. Chem. Soc. 117: 1181-1182.

* Pornillos O, Ganser-Pornillos BK, Kelly BN, Hua Y, Whitby FG, Stout CD, Sundquist WI, Hill CP, Yeager M. 2009. [X-ray structures of the hexameric building block of the HIV capsid](https://dx.doi.org/10.1016/j.cell.2009.04.063). Cell 137: 1282-1292.

* Hill CP, Worthylake D, Bancroft DP, Christensen AM, Sundquist WI. 1996. [Crystal structures of the trimeric human immunodeficiency virus type 1 matrix protein: implications for membrane association and assembly](http://www.ncbi.nlm.nih.gov/pubmed/8610175). Proc. Natl. Acad. Sci. 93: 3099-3104.

* Pancera M, Zhou T, Druz A, Georgiev IS, Soto C, Gorman J, Huang J, Acharya P, Chuang GY, Ofek G, Stewart-Jones GB, Stuckey J, Bailer RT, Joyce MG, Louder MK, Tumba N, Yang Y, Zhang B, Cohen MS, Haynes BF, Mascola JR, Morris L, Munro JB, Blanchard SC, Mothes W, Connors M, Kwong PD. 2014. [Structure and immune recognition of trimeric pre-fusion HIV-1 Env](https://dx.doi.org/10.1038/nature13808). Nature 514: 455-461.

<!-- Basic Tools -->
* Katoh K, Frith MC. 2012. [Adding unaligned sequences into an existing alignment using MAFFT and LAST](https://dx.doi.org/10.1093/bioinformatics/bts578). Bioinformatics 28: 3144-3146.

* Katoh K, Standley DM. 2013. [MAFFT multiple sequence alignment software version 7: improvements in performance and usability](https://dx.doi.org/10.1093/molbev/mst010). Mol. Biol. Evol. 30: 772-780.

* Katoh K. Standley DM. 2014. [MAFFT: iterative refinement and additional methods](https://dx.doi.org/10.1007/978-1-62703-646-7_8). Methods in Molecular Biology 1079: 131-146.

<!-- Structural Constraints -->
* Scherrer MP, Meyer AG, Wilke CO. 2012. [Modeling coding-sequence evolution within the context of residue solvent accessibility](https://dx.doi.org/10.1186/1471-2148-12-179). BMC Evol. Biol. 12: 179.

* Meyer AG, Wilke CO. 2012. [Integrating sequence variation and protein structure to identify sites under selection](http://dx.doi.org/10.1093/molbev/mss217). Mol. Biol. Evol. 30: 36-44.

* Meyer AG, Dawson ET, Wilke CO. 2013. [Cross-species comparison of site-specific evolutionary-rate variation in influenza hemagglutinin](http://dx.doi.org/10.1098/rstb.2012.0334). Phil. Trans. R. Soc. B 368: 20120334.

* Meyer AG, Wilke CO. 2015. [Geometric Constraints Dominate the Antigenic Evolution of Influenza H3N2 Hemagglutinin](http://dx.doi.org/10.1371/journal.ppat.1004940). PLOS Pathogens 11: e1004940.

<!-- Basic Tools -->
* Kabsch W, Sander C. 1983. [Dictionary of protein secondary structure: pattern recognition of hydrogen-bonded and geometrical features](http://www.ncbi.nlm.nih.gov/pubmed/6667333). Biopolymers 22: 2577-2637.

* Tien MZ, Meyer AG, Sydykova DK, Spielman SJ, Wilke CO. 2013. [Maximum allowed solvent accessibilites of residues in proteins](https://dx.doi.org/10.1371/journal.pone.0080635). PLOS ONE 8: e80635.

* Ihaka R, Gentleman R. 1996. [R: A language for data analysis and graphics](https://dx.doi.org/10.1080/10618600.1996.10474713). Journal of Computational and Graphical Statistics 5: 299-314.

* Wickham H. 2009. [ggplot2: elegant graphics for data analysis](http://had.co.nz/ggplot2/book).

<!-- Structural Constraints -->
* Bustamante CD, Townsend JP, Hartl DL. 2000. [Solvent Accessibility and Purifying Selection Within Proteins of Escherichia coli and Salmonella enterica](http://www.ncbi.nlm.nih.gov/pubmed/10677853). Mol. Biol. Evol. 17: 301-308.

* Franzosa EA, Xia Y. 2009. [Structural Determinants of Protein Evolution Are Context-Sensitive at the Residue Level](https://dx.doi.org/10.1093/molbev/msp146). Mol. Biol. Evol. 26: 2387-2395.

* Ramsey DC, Scherrer MP, Wilke CO. 2011. [The relationship between relative solvent accessibility and evolutionary rate in protein evolution](https://dx.doi.org/10.1534/genetics.111.128025). Genetics 188: 479-488.

* Shahmoradi A, Sydykova DK, Spielman SJ, Jackson EL, Dawson ET, Meyer AG, Wilke CO. 2014. [Predicting evolutionary site variability from structure in viral proteins: buriedness, packing, flexibility, and design](https://dx.doi.org/10.1007/s00239-014-9644-x). J. Mol. Evol. 79: 130-142.

* Echave J, Jackson EL, Wilke CO. 2014. [Relationship between protein thermodynamic constraints and variation of evolutionary rates among sites](https://dx.doi.org/10.1088/1478-3975/12/2/025002). Phys. Biol. 12: 025002.

* Marcos M, Echave J. 2015. [Too packed to change: side-chain packing and site-specific substitution rates in protein evolution](https://dx.doi.org/10.7717/peerj.911). PeerJ 3: e911.
  
* Huang TT, Marcos ML, Hwang JK, Echave J. 2014. [A mechanistic stress model of protein evolution accounts for site-specific evolutionary rates and their relationship with packing density and flexibility](https://dx.doi.org/10.1186/1471-2148-14-78). BMC Evol. Biol. 78: 14.

* Sikosek T, Chan HS. 2014. [Biophysics of protein evolution and evolutionary protein biophysics](https://dx.doi.org/10.1098/rsif.2014.0419). J. Royal Soc. Interface 11: 20140419.

* Grahnen JA, Nandakumar P, Kubelka J, Liberles DA. 2011. [Biophysical and structural consideration for protein sequence evolution](https://dx.doi.org/10.1186/1471-2148-11-361). BMC Evol. Biol. 11: 2455-2464.

<!-- HIV Papers -->
* Mansky LM, Temin HM. 1995. [Lower in vivo mutation rate of human immunodeficiency virus type 1 than that predicted from the fidelity of purified reverse transcriptase](http://www.ncbi.nlm.nih.gov/pubmed/7541846). J. Virol. 69: 5087-5094.

* Frost SD, Wrin T, Smith DM, Kosakovsky Pond SL, Liu Y, Paxinos E, Chappey C, Galovich J, Beauchaine J, Petropoulos CJ, Little SJ, Richman DD. 2005. [Neutralizing antibody responses drive the evolution of human immunodeficiency virus type 1 envelope during recent HIV infection](https://dx.doi.org/10.1073/pnas.0504658102). Proc. Natl. Acad. Sci. 102: 18514–18519.

* Wei X, Ghosh SK, Taylor ME, Johnson VA, Emini EA, Deutsch P, Lifson JD, Bonhoeffer S, Nowak MA, Hahn BH, Saag MS, Shaw GM. 1995. [Viral Dynamics in human immunodeficiency virus type 1 infection](https://dx.doi.org/10.1038/373117a0) Nature 373: 117-122.

* Wei X, Decker JM, Wang S, Hui H, Kappes JC, Wu X, Salazar-Gonzalez JF, Salazar MG, Kilby JM, Saag MS, Komarova NL, Nowak MA, Hahn BH, Kwong PD, Shaw GM. 2002. [Antibody neutralization and escape by HIV-1](https://dx.doi.org/10.1038/nature01470) Nature. 422: 307-12.

* Perelson AS, Neumann AU, Markowitz M, Leonard JM, Ho DD. 1996. [HIV-1 dynamics in vivo: Virion clearance rate, infected cell life-span, and viral generation time](https://dx.doi.org/10.1126/science.271.5255.1582). Science 271: 1582-1586.

* Perelson AS, Essunger P, Cao Y, Vesanen M, Hurley A, Saksela K, Markowitz M, Ho DD 1997. [Decay characteristics of HIV-1-infected compartments during combination therapy](https://dx.doi.org/10.1038/387188a0). Nature 387: 188-191.

* Chun TW, Carruth L, Finzi D, Shen XF, DiGiuseppe JA, Taylor H, Hermankova M, Chadwick K, Margolick J, Quinn TC, Kuo YH, Brookmeyer R, Zeiger MA, BarditchCrovo P, Siliciano RF. 1997. [Quantification of latent tissue reservoirs and total body viral load in HIV-1 infection](https://dx.doi.org/10.1038/387183a0). Nature 387: 183-188.














